.github/workflows/widget_test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Unit_Test
```

Comment:
Is it possible to use spaces in the name? Like "Unit tests". This way it'll look human readable when we add a status badge.

See https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository

Reply:
It looks like it's indeed possible: https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#workflow-file-example